### ISSUE TITLE
wont save courses 

### ISSUE THREAD

**Issue by gravlar93** [Courseplay/courseplay#1349]
hello I'm using 4.01 and I'm trying to save courses they work when I record them in game but when I leave game and go back on game they just disappear?

**Comment by Satissis** [comment]
Update to the latest developing version which is 4.01.0119 and check if that fixes it.

**Comment by gravlar93** [comment]
Hello that didn't work

**Comment by Satissis** [comment]
Could you post log on Gist and link it here ?

**Comment by Satissis** [comment]
Once you have played, with this problem, then you have an log.txt file in: C:\Users\[YOUR USER NAME]\Documents\My Games\FarmingSimulator2015 (In windows)
Copy the text in that file and goto Gist: https://gist.github.com/
Past in the log and press the create public gist.
Then copy the url from that page and post it here, that way we can see if there is any errors in the game log file that could lead to this problem.

**Comment by gravlar93** [comment]
Hello I've fixed the problem I looked on the log document and there was some mods with errors so deleted them and now course play save courses, thanks for your help

Regards

Ryan

Sent from my iPhone

On 7 Oct 2016, at 13:51, Claus G. Pedersen <notifications@github.com<mailto:notifications@github.com>> wrote:


Once you have played, with this problem, then you have an log.txt file in: C:\Users[YOUR USER NAME]\Documents\My Games\FarmingSimulator2015 (In windows)
Copy the text in that file and goto Gist: https://gist.github.com/
Past in the log and press the create public gist.
Then copy the url from that page and post it here, that way we can see if there is any errors in the game log file that could lead to this problem.
